### PR TITLE
String OM fix for 1145

### LINKED ISSUE #1145: Qt OMs contains duplicate of C++ standard abstractions
Both the QtCore and QtGui folders contain an implementation of `string`. The implementation is different from the main abstract model (at cpp/library).

Is there any reason for those headers (a Qt only specific optimization) or we can just remove it?

### PR DESCRIPTION
**Pull request by kunjsong01**
Fixed https://github.com/esbmc/esbmc/issues/1145 to avoid interference with `cpp/library/string`

Just renamed it for future reference, just in case we want to run the qt test suite again. 
I'm also OK with removing them if required.

### CODE REVIEW

**Comment by rafaelsamenezes** [comment]
Why don't just remove it? 

If you find that it broke something you can revert the PR.

**Comment by kunjsong01** [comment]
> Why don't just remove it?
> 
> If you find that it broke something you can revert the PR.

removed

File: regression/esbmc-cpp/OM_sanity_checks/ciso646/main.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include <cisco646>
+#include <ciso646>
```
Comment (rafaelsamenezes) [inline regression/esbmc-cpp/OM_sanity_checks/ciso646/main.cpp]: Why this file is part of the PR?

**Review by rafaelsamenezes — CHANGES_REQUESTED** [review]

File: regression/esbmc-cpp/OM_sanity_checks/ciso646/main.cpp (hunk 1 above)
Comment (kunjsong01) [reply regression/esbmc-cpp/OM_sanity_checks/ciso646/main.cpp]: removed

**Review by rafaelsamenezes — APPROVED** [review]

**Review by lucasccordeiro — APPROVED** [review]

**Comment by lucasccordeiro** [comment]
Thanks for submitting this PR, @kunjsong01.

**Pull request merged**